src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -579,7 +583,7 @@ subroutine InitHistory(this, bounds)
        this%rc13_canair_patch(begp:endp) = spval
        call hist_addfld1d (fname='RC13_CANAIR', units='proportion', &
             avgflag='A', long_name='C13/C(12+13) for canopy air', &
-            ptr_patch=this%rc13_canair_patch, default='inactive')
+            ptr_patch=this%rc13_canair_patch, set_spec=spval, default='inactive')
```

Comment:
Nit: Organize all c13 calls under one if-statement for computational efficiency?